ai21/clients/common/conversational_rag.py

```diff
@@ -0,0 +1,44 @@
+from abc import ABC, abstractmethod
+from typing import Any, Dict, List
+
+from ai21.models import ChatMessage
+from ai21.models.responses.conversational_rag_response import ConversationalRagResponse
+from ai21.models.retrieval_strategy import RetrievalStrategy
+from ai21.types import NotGiven
+
+
+class ConversationalRag(ABC):
+    _endpoint = "/studio/v1/conversational-rag"
```

Comment:
Put down only `conversational-rag`
The url is populated automatically by the client